public/app/redux/reducers/settings.ts

```diff
@@ -88,7 +88,8 @@ export const reloadUsers = createAsyncThunk(
 export const reloadApps = createAsyncThunk(
   'newRoot/reloadApps',
   async (_, thunkAPI) => {
-    const res = await fetchApps();
+    // TODO(bryan) What is this?
+    const res = await fetchApps(0, 0);
```

Comment:
I'm unsure how to get the start and end here. In fact, I don't really know where in the UI a trigger for this reducer lives.